preorder/src/App.js

```diff
@@ -302,6 +302,7 @@ class ShippingForm extends Component {
 								<option value="KE">Kenya</option>
 								<option value="KI">Kiribati</option>
 								<option value="KS">Korea South</option>
+								<option value="KS">Kosovo, Republic Of</option>
```

Comment:
Kosovo and South Korea have the same country code here, shouldn't they be different?